login.py

```
@@ -4,7 +4,10 @@
 def login(username, password):
     conn = sqlite3.connect("users.db")
     cursor = conn.cursor()
-    query = f"SELECT * FROM users WHERE username='{username}' AND password='{password}'"
+    query = f"SELECT * FROM users WHERE username='Aarti' AND password='Bhosale'"
     cursor.execute(query)
     user = cursor.fetchone()
-    return user
+    print (
+        testing
```

Ruff (0.14.6)

Undefined name `testing`

(F821)